config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   get '/users/:user_id/posts', to: 'posts#index', as: 'post_path'
   get '/users/:user_id/posts/:id', to: 'posts#show', as: 'post_show'
-
+  post '/users/:user_id/posts/:id/comments', to: 'comments#create', as: 'comment_create'
```

Comment:
Great job @taldr27 🤝

- [ ] There should be a pair of routes in the config/routes.rb file for creating comments.
![comment-route](https://user-images.githubusercontent.com/63917725/216448647-f2859324-ec90-4cf6-ac19-c54c43c30e65.PNG)

- [ ] There should be a form in the views/comments/new.html.erb file which accepts input for creating comments.

Reply:
@1GORDON Hello, thanks for your review, I have a question to solve this problem, do I have to create a new page, like with a new post?

Reply:
Hi @taldr27 ,
- Just like you created a form for posts. You can also create a form form comments.